src/arch/riscv/isa/templates/vector_mem.isa

```diff
@@ -137,6 +137,8 @@ Fault
         return std::make_shared<IllegalInstFault>(
             "RVV is disabled or VPU is off", machInst);
     }
+    if (machInst.vill)
+        return std::make_shared<IllegalInstFault>("VILL is set", machInst);
```

Comment:
Just curious... why do you raise this exception in execute instead of decode?

Reply:
"Any vector instructions depended on vtype will raise illegal instruction" according to [spec](https://github.com/riscv/riscv-v-spec/blob/master/v-spec.adoc#344-vector-type-illegal-vill). IMHO, it's OK for me to put it in either decode or execute. However, in the gem5 model, we can only raise illegal instruction in execute, just put them in execute.

Reply:
Got it. Thanks!